<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="blurOrigGray">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gråtone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="garden">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Garden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocean">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ocean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stillife">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stillife</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blurColour">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Glatt</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="constant">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="blurOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Glatting av bilde</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mountains">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mountains</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Legger først litt støy på &lt;br/&gt;bildet, og glatter deretter&lt;br/&gt;bildet ved hjelp av poisson-&lt;br/&gt;likningen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Velg mellom å glatte et &lt;br/&gt;fargebilde eller et gråtone-&lt;br/&gt;bilde. Ved å sette verdien&lt;br/&gt;høyere vil bildet bli mindre&lt;br/&gt;glattere.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="balls">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adjuster">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adjuster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blurCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fog">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trees">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blurGray">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Glatt (grå)</string>
   </property>
  </widget>
  <widget class="QLabel" name="blurImg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>76</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Adjuster/Adjuster_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
